src/Core/Models/Data/Provider/ProviderOrganizationUnassignedOrganizationDetails.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Bit.Core.Enums;
+
+namespace Bit.Core.Models.Data;
+
+public class ProviderOrganizationUnassignedOrganizationDetails
```

Comment:
Suggestion: we can simplify this to `ProviderOrganizationUnassignedDetails` without losing any meaning.

Although I'm also not sure about the `ProviderOrganization` naming and folder. There is no `ProviderOrganization` association here (because they're unassigned), and this is really just a subset of Organization details (no Provider data).

Maybe `OrganizationWithOwnerDetails` instead?
Or if you still want to communicate what it's used for - `OrganizationWithoutProviderDetails`?

Any ideas?

Reply:
UPDATE: see my other comment first, this might be irrelevant: https://github.com/bitwarden/server/pull/2683/files#r1112568487

Reply:
I've deleted this class and went on to just use `Organization` because I removed the owner emails column from the result. If Product decides to keep the column then I'll create a class called `OrganizationWithOwnerDetails`

Reply:
Update: We can remove the emails column